### PR TITLE
Moving Router to a specific node

### PR DESCRIPTION
**Pull request by v2pkthakur**
added commands to move the router to a specific labeled node.

### CODE REVIEW

File: content/routers/_index.md [hunk 1]
```diff
@@ -16,3 +16,19 @@ oc patch \
    --type=merge \
    ingresscontroller/default
 ```
+
+# Moving routers to a specific (infra) node 
+
+Label the desired node with a particular label (infra)
```
Comment (e-minguez) [inline content/routers/_index.md:22]:
```suggestion
Label the desired nodes with a particular label (infra) and add a taint to those nodes:
```

File: content/routers/_index.md [hunk 2]
```diff
@@ -16,3 +16,19 @@ oc patch \
...
+```
+oc label node worker1  node-role.kubernetes.io/infra=
+oc label node worker2  node-role.kubernetes.io/infra=
+oc adm taint nodes -l node-role.kubernetes.io/infra infra=reserved:NoSchedule infra=reserved:NoExecute
+```
+
+Patch the ingresscontroller to use the nodes with specific nodeselector
```
Comment (e-minguez) [inline content/routers/_index.md:29]:
```suggestion
Patch the `ingresscontroller` to use the nodes with specific `nodeselector` and a toleration for the previous `taint`:
```

**Review by e-minguez — CHANGES_REQUESTED** [review]
Thanks for your PR. I've added some suggestions :)

File: content/routers/_index.md (hunk 2 above)
Comment (v2pkthakur) [reply content/routers/_index.md:29]: I am good. Thanks for adding more explanation.

File: content/routers/_index.md (hunk 1 above)
Comment (v2pkthakur) [reply content/routers/_index.md:22]: I am good. Thanks for adding more explanation.

Comment (e-minguez) [reply content/routers/_index.md:22]: Would you mind to add those changes? Thanks!

File: content/routers/_index.md (hunk 2 above)
Comment (e-minguez) [reply content/routers/_index.md:29]: Would you mind to add those changes? Thanks!

**Review by e-minguez — CHANGES_REQUESTED** [review]

**Pull request merged**